noir_stdlib/src/field.nr

```diff
@@ -0,0 +1,26 @@
+#[builtin(to_bits)]
+fn to_bits(_x : Field, _bit_size: u32) -> [u1] {}
```

Comment:
It is a bit odd to me that we have `to_bits` and `to_le_bytes`. What order are the bits in?

Reply:
This is odd it should be renamed, I'll do it in this PR. They are little-endian. Might want to add another builtin for `to_be_bits` as well